CHANGELOG.md

```diff
@@ -30,6 +38,7 @@ For detailed information on the tasks carried out during this release, please se
 For detailed information on the tasks carried out during this release, please see the GitHub milestone
 [v5.10.0.1](https://github.com/wso2/docker-is/milestone/10).
 
+[v5.10.0.4]: https://github.com/wso2/docker-is/compare/v5.9.0.3...v5.10.0.4
```

Comment:
@NishikaDeSilva please correct the [link](https://github.com/wso2/docker-is/compare/v5.9.0.3...v5.10.0.4).

Reply:
fixed